pkg/istio/rbac/common/util.go

```diff
@@ -96,6 +98,24 @@ func (a *ApiHandler) Delete(item *Item) error {
 	return err
 }
 
+// CheckIfMemberIsAllUsers returns namespace for Athenz domain when role member is of form '<athenz-domain>.*'.
+// Example: domain.sub-domain.* -> domain-sub--domain
+func CheckIfMemberIsAllUsers(member *zms.RoleMember, domainName zms.DomainName) (string, error) {
+
+	if member == nil {
+		return "", fmt.Errorf("member is nil")
+	}
+
+	memberStr := string(member.MemberName)
+
+	// if member name is of the form '<athenz-domain>.*', return namespace
+	if strings.HasPrefix(memberStr, "user.") || !strings.HasSuffix(memberStr, ".*") {
+		return "", nil
+	}
+
+	return athenz.DomainToNamespace(memberStr[0 : len(memberStr)-2]), nil
```

Comment:
you can avoid string parsing by just calling athenz.DomainToNamespace(domainName)

Reply:
We can't use `domainName` here because that's the domain of the role whose members are currently getting processed, and all members won't necessarily belong to the same domain.

Reply:
Makes sense